docs/concepts/Alexa-Hosted-Skill-Commands.md

```diff
@@ -40,10 +40,11 @@ The user is not prompted for a `skill template`. The [Hello World Skill](https:/
 
 This command initializes Alexa Hosted Skills by cloning the project from the hosted skill service, downloading the latest Alexa skill package, and provide a git-ready environment. Developers can then checkout, pull from, and push to a remote Git repository from their local machines.
 
+If you make changes to the skill package such as the interaction model or publishing information on the Alexa Developer Console, these changes will not be reflected in your git repository. You will have to resync the skill-package which you could you do by running `ask init --hosted-skill-id <id> --export-package` and committing the changes that were made on the console.
```

Comment:
This seems conceptually more like a "sync-package" operation to me.. When I think of "import" I think of the user specifying a location to grab the skill-package from, and "export" would be like dumping the package to a location the user specifies. Am I understanding right that this is just synchronizing the state between the skill stored in the developer console and the local skill package?

Reply:
Yeah I agree, I used `export-package` since there's a command

```
ask smapi export-package
```

which downloads it from smapi. (I guess name was chosen since it's exporting it from UI)

`sync-package` is good for more clearness, although it current approach it doesn't necessarily `sync` them since it doesn't commit the changes

Reply:
Yeah that's fair. They'll at least be listed as modified files so they'll (likely) get committed at some point. Maybe a point to expand on this in the future, we could stash any local changes, sync with the smapi 'export-package', commit, then pop the stash to re-apply any outstanding local changes?